docker/ccd-definition-import/scripts/template_ccd_definition.py

```diff
@@ -10,21 +10,20 @@
 template_file_location = sys.argv[1]
 output_file_location = sys.argv[2]
 
-bulk_scan_orchestrator_base_url = os.environ['BULK_SCAN_ORCHESTRATOR_BASE_URL']
-
-wb = load_workbook(template_file_location)
-bulk_scan_template = '${BULK_SCAN_ORCHESTRATOR_BASE_URL}'
+service_base_url_placeholder = '${BULK_SCAN_ORCHESTRATOR_BASE_URL}'
+service_base_url = os.environ['BULK_SCAN_ORCHESTRATOR_BASE_URL']
 
 def template_sheet(ws, callback_columns):
   print('Updating CCD sheet ' + ws.title)
   count_of_cells_updated = 0
   for column in callback_columns:
     for cell in ws[column]:
-      if cell.value is not None and bulk_scan_template in cell.value:
-        cell.value = cell.value.replace(bulk_scan_template, bulk_scan_orchestrator_base_url)
+      if cell.value is not None and service_base_url_placeholder in cell.value:
+        cell.value = cell.value.replace(service_base_url_placeholder, service_base_url)
         count_of_cells_updated += 1
   print('Successfully updated BSO %s, %d cells were updated' % (ws.title, count_of_cells_updated))
 
+wb = load_workbook(template_file_location)
 case_event_ws = wb['CaseEvent']
 case_event_callback_columns = ['J', 'L', 'N']
 template_sheet(case_event_ws, case_event_callback_columns)
```

Comment:
I intend to inline this function on next PR

Reply:
that's there because previously it was templating multiple sheets, but then the need was removed, up to you but there is at least one more sheet that can have callbacks the one about printing the case out